app/src/main/kotlin/com/brianco/simpledaycounter/DayCounterWidgetConfigureActivity.kt

```diff
@@ -78,26 +78,18 @@ class DayCounterWidgetConfigureActivity : Activity() {
       "#8BC34A",
       "#FFA726"
     )
-    val backgroundColorLinearLayoutView = findViewById<LinearLayout>(R.id.background_color_circle_container)
+    val backgroundColorLinearLayoutView = findViewById<LinearLayout>(
+      R.id.background_color_circle_container
+    )
     backgroundColors.forEach { colorHex ->
       val color = Color.parseColor(colorHex)
       addColorCircle(color, backgroundColorLinearLayoutView, true)
     }
 
-    datePicker.setOnDateChangedListener(object : OnDateChangedListener {
-      override fun onDateChanged(
-        view: DatePicker,
-        year: Int,
-        monthOfYear: Int,
-        dayOfMonth: Int,
-      ) {
-        val dayCount = daysSince(datePicker.dateMidnightUtcMillis(), System.currentTimeMillis(), TimeZone.getDefault()).run {
-          check(this <= Integer.MAX_VALUE)
-          toInt()
```

Comment:
this never needed to force the day count to be an 4-byte Int. let's keep it as a Long.